### ISSUE TITLE
Links to the budget library for new countries

### ISSUE THREAD

**Issue by pwalsh** [okfn/ibp-explorer#76]
# Client requests
- Each country page contains links to “View the Budget Library” and “View historical information.” The budget library button links to the country page on our Google Drive, which hosts budget documents. With the 75 countries that get added, we’ll need to link those country pages to the Google Drive.
- **Further explanation on above**: If you look at each country page in the current Tracker site, you see there are two blue buttons—one linking to the historical info, and a second one linking to the budget library. It would be ideal to add that link to the library for all countries, but maybe we can discuss what the additional cost would be, and take it from there? 
# We need to know
- @dumyan please check this

**Comment by dumyan** [comment]
Currently the library URL is provided with the API. But since we have created a GDrive tree parser we can easily map new countries with the IDs we have in the spreadsheet. In essence, this will work exactly as the document and document group links we have implemented in the tracker.

**Comment by pwalsh** [comment]
Status: Client is working on a strict naming convention for file sin library. with this, we will be able to use our implementation to link to files.

**Comment by dumyan** [comment]
We have now implemented the google drive parser which is used to match documents according to their path in the drive. Currently because I've got better results using the filename searching and the new path searching, we have them both in place. When the client makes all the changes to google drive we can delete the direct filename mapping and rely only on the path. The changes will require some 10 minutes as the mapping in done in one function.
